src/covid_model_seiir_pipeline/forecasting/model/forecast_metrics.py

```diff
@@ -89,17 +90,17 @@ def _cleanup(infections: pd.Series) -> pd.DataFrame:
                 .sort_index()['infections'])
 
     susceptible_columns = [c for c in components.columns if 'S' in c]
-    exposed_and_protected_columns = [c for c in components.columns if 'E_p' in c]
+    newE_protected_columns = [c for c in components.columns if '_p' in c and 'S' not in c]
```

Comment:
`newE` is ... new exposures?

Reply:
New exposures among those in the protected group.

Reply:
I'll add some comments.  This is a confusing thing.